wunderio/next4drupal-project
# NEX-80: File attachments paragraph

This PR adds:

## Drupal:
1. a new paragraph called "File attachments" to which multiple media files can be attached. The paragraph can be added to frontpage, page, and inside accordions.
2. Migrations to add this component to the demo content (test pdf included)
(Due to a limitation of how we have the source files for the migration, only one file per paragraph is added... but the paragraph can accept any number of files and the frontend is aware of it)

## Next
1. New paragraph
2. Some changes about how we render the media items inside paragraphs (now using the generic <Media> component, which then renders the correct component based on the type of media passed.
3. A basic frontend listing of files, with a (fixed icon), file name - link , mime type and formatted size

### Screenshots

![image](https://github.com/wunderio/next4drupal-project/assets/185412/4f3da0c6-f11e-4930-9a9f-8ac004c3c2cd)


### To test

1. Visit the "Media and publishing" page, see the files displayed.
2. add some files, add a component to a page, see the files displayed

On Silta, this is the relevant page: 
(note that there is only one file as part of the migration (because reasons), but more can be added if you log into the drupal side and edit the content)

https://feature-nex-80-next.next4drupal-project.dev.wdr.io/en/media-and-publishing
